azure/errors.go

```diff
@@ -0,0 +1,9 @@
+package azure
+
+const (
+	publishSettingsConfigurationError = "PublishSettingsFilePath is set. Consequently ManagementCertificatePath and SubscriptionId must not be set."
+
+	managementCertificateConfigurationError = "ManagementCertificatePath is set. Consequently SubscriptionId must also be set, and PublishSettingsFilePath must not be set."
```

Comment:
This needs more attention, it's completely untested with anything except a publish settings file. Also it needs a better way to choose a subscription from a file with more than one.